src/app/frontend/logs/component.js

```diff
@@ -176,6 +183,7 @@ export class LogsController {
    * @export
    */
   toggleLogFollow() {
+    this.toggleIntervalFunction_();
     this.logsService.setFollowing();
```

Comment:
By swapping this line with toggling interval we can get rid of below if statement. Logs will be reloaded immediately after first interval start.